core/cas-server-core-authentication-mfa/src/main/java/org/apereo/cas/authentication/GroovyMultifactorAuthenticationProviderBypass.java

```diff
@@ -49,6 +49,7 @@ public boolean shouldMultifactorAuthenticationProviderExecute(final Authenticati
         } catch (final Exception e) {
             LOGGER.error(e.getMessage(), e);
         }
-        return super.shouldMultifactorAuthenticationProviderExecute(authentication, registeredService, provider, request);
+        LOGGER.error("Error encountered executing groovy bypass, returning shouldExecute true as default");
```

Comment:
This probably needs to be reworded.

Reply:
Ah never mind. I missed the fact that true does actually activate MFA which is correct behavior. So that's good; just reword the log message, for this and the REST one likely.